### ISSUE TITLE
Add Coverage

### ISSUE THREAD

**Issue by jsug9** [Tadesse-Alemayehu/To-Do-list#11]
https://github.com/Tadesse-Alemayehu/To-Do-list/blob/95908beeab983a5459ba3a9b5ab7605e98812eab/package.json#L7

Good job implementing jest, but I advise you to also add coverage to see how much of your code is being tested:
```
"coverage": "jest --coverage",
```
If you are using jsdom, just add it:
```
"coverage": "jest --env=jsdom --coverage",
```

**Comment by TadesseDev** [comment]
@jsug9 Thank you brother 🙏🏼. We will make the changes 🙌

**Comment by TadesseDev** [comment]
@jsug9 this issue have been fixed at 0b56b9d464d0a8bd52a23614594714c7fce03d49 commit hash. Thank you for the reminder.
